apps/server/src/modules/room/api/room.uc.ts

```diff
@@ -113,6 +115,13 @@ export class RoomUc {
 		await this.roomMembershipService.deleteRoomMembership(roomId);
 	}
 
+	public async duplicateRoom(userId: EntityId, roomId: EntityId): Promise<void> {
+		this.checkFeatureEnabled();
+		this.checkFeatureRoomsDuplicationEnabled();
+		await this.checkRoomAuthorizationByIds(userId, roomId, Action.write, [Permission.ROOM_DUPLICATE]);
+		//TODO Service is missing
+	}
```

Comment:
At this point we could just return the original `roomId`. This could then be used in the Frontend to just redirect to the original room details.